tests/collectors/test_ioos_swe_sos.py

```diff
@@ -19,19 +21,22 @@ def setUp(self):
         # self.c.features = [urn.name for urn in self.c.server.offerings]
 
 
+    @pytest.mark.xfail
```

Comment:
The endpoint http://data.nanoos.org/52nsos/sos/kvp is timing out (`504 Gateway Time-out`). Ping @emiliom who may know that is going on here. Skipping the tests for now...

Reply:
We had a problem this morning with a server involved in the SOS service. It's fixed. 

But more generally, maybe this test should be changed to point to a different SOS service?

Reply:
Thanks for the info @emiliom. In theory all the unittest should be mocked and not relying on remote services, only integration tests should rely on those and should be `xfail`. However, that is a considerable amount of work and I'm not sure we can afford to do that for `pyoos` (unless there is a community interest to keep this library alive).